Skype/WebSDK/docs/TermsOfService.md

```diff
@@ -1,3 +1,8 @@
+---
+title: MICROSOFT SOFTWARE LICENSE TERMS
```

Comment:
Please take the all-caps off here. I would make this: "Microsoft software license terms for Skype Web SDK."